app/views/provider/admin/dashboards/widgets/_service_navigations.html.slim

```diff
@@ -1,75 +1,76 @@
-nav.DashboardNavigation
-  ul.DashboardNavigation-list
+div id=widget.id
```

Comment:
What's this for?

Reply:
otherwise, the widget content is not being replaced.